conf/application.conf

```diff
@@ -200,6 +201,10 @@ microservice {
         with-js = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/problem_reports_ajax?service=AMLS"
         non-js = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/problem_reports_nonjs?service=AMLS"
       }
+      accessibility-statement-url {
+        authenticated = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/accessibility"
+        unauthenticated = ${microservice.services.contact-frontend.protocol}"://"${microservice.services.contact-frontend.host}":"${microservice.services.contact-frontend.port}"/contact/accessibility"
```

Comment:
We will need to look at adding these to app-config-base also. Need a PR for this.

Reply:
Done